src/prplmesh_getall.lua

```diff
@@ -402,9 +402,15 @@ local function get_mmx_out(path)
 
     dfs(root)
 
+    --[[
+        If MMX output string is empty need to return empty list of indexes
+        to the MMX Core which contains just successfull operation code: "0;".
+        This logic gives MMX ability to clean Data Base when indexes do not
+        exist for the given path.
+    --]]
     if string.len(mmx_out_str) == 0 then
         error("Empty output string.")
-        return false
+        return tostring(ing.ResCode.SUCCESS) .. ";"
     end
 
     local mmx = tostring(ing.ResCode.SUCCESS) .. ";" .. mmx_out_str
```

Comment:
Instead of lines 411-418 (of new revision):
```
return tostring(ing.ResCode.SUCCESS) .. ";" .. mmx_out_str
```
We don't need to print next error message because we don't decide it an error more
```
tostring(ing.ResCode.SUCCESS) .. ";" .. mmx_out_str
```
as a result we don't need to check length of `mmx_out_str` and etc.